### PR TITLE
accelerate MIMIC-III data loading by pandarallel

### PR DESCRIPTION
**Pull request by ycq091044**
MIMIC-III table loading test:

Before:
- PATIENTS and ADMISSIONS: 31s
- PRESCRIPTIONS: 3min17s
- LABEVENTS: 17min27s
- DIAGNOSIS_ICD: 3s (something is wrong here?)
- PROCEDURES_ICD: 2s (something is wrong here?)

After:
- PATIENTS and ADMISSIONS: 6.29s
- PRESCRIPTIONS: 59.7s
- LABEVENTS: 298.49s
- DIAGNOSIS_ICD: 12s
- PROCEDURES_ICD: 10s

---
Add some modifications on `example/` for testing v1.1.3 version

### CODE REVIEW

File: examples/test.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import sys
+
+sys.path.append("/home/chaoqiy2/github/PyHealth")
```
Comment (zlin7) [inline examples/test.py]: I think this file name and this path should be changed if you want it in `develop`. Or maybe we will remove this file altogether later?

Can use something like `os.path.dirname(os.path.abspath(__file__))` if necessary

File: pyhealth/datasets/base_dataset.py [hunk 2]
```diff
@@ -4,9 +4,9 @@
 from collections import Counter
 from copy import deepcopy
 from typing import Dict, Callable, Tuple, Union, List, Optional
-
+import pandas as pd
 from tqdm import tqdm
-
+import time
```
Comment (zlin7) [inline pyhealth/datasets/base_dataset.py]: The import orders should probably be fixed

File: pyhealth/datasets/mimic3.py [hunk 3]
```diff
@@ -3,11 +3,13 @@
 
 import pandas as pd
 from tqdm import tqdm
-
+import time
```
Comment (zlin7) [inline pyhealth/datasets/mimic3.py]: import order

File: examples/test.py [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+from pyhealth.models import Transformer
+
+device = "cuda:0"
+model = Transformer(
+    dataset=dataset,
+    feature_keys=["conditions", "procedures"],
+    label_key="label",
+    mode="binary",
+    operation_level="visit",
```
Comment (zlin7) [inline examples/test.py]: I failed here. operation_level is removed?

File: pyhealth/datasets/mimic3.py [hunk 5]
```diff
@@ -150,24 +161,37 @@ def parse_diagnoses_icd(self, patients: Dict[str, Patient]) -> Dict[str, Patient
             os.path.join(self.root, f"{table}.csv"),
             dtype={"SUBJECT_ID": str, "HADM_ID": str, "ICD9_CODE": str},
         )
+        # drop records of the other patients
+        df = df[df["SUBJECT_ID"].isin(patients.keys())]
         # drop rows with missing values
         df = df.dropna(subset=["SUBJECT_ID", "HADM_ID", "ICD9_CODE"])
         # sort by sequence number (i.e., priority)
-        df = df.sort_values(["SUBJECT_ID", "HADM_ID", "SEQ_NUM"], ascending=True)
+        df = df.sort_values(["SUBJECT_ID", "HADM_ID", "ICD9_CODE"], ascending=True)
```
Comment (ycq091044) [inline pyhealth/datasets/mimic3.py]: change ICD9_CODE back to SEQ_NUM

File: examples/test.py (hunk 4 above)
Comment (ycq091044) [reply examples/test.py]: I see. This test file is an very old one.

**Pull request merged**